rackcity/utils/errors_utils.py

```diff
@@ -15,11 +16,14 @@ class Status(Enum):
 
 
 class GenericFailure(Enum):
-    UNKNOWN = "Unknown internal error."
+    INTERNAL = "Internal error."
```

Comment:
this is used SPARINGLY only when `id` is not specified in a request or it is specified when it shouldn't be